packages/lesswrong/components/ea-forum/voting-portal/EAVotingPortalSelectCandidatesPage.tsx

```diff
@@ -89,7 +90,7 @@ const EAVotingPortalSelectCandidatesPage = ({
               await saveSelection();
               navigate({ pathname: "/voting-portal/compare" });
             },
-            disabled: selectedIds.length === 0
+            disabled: selectedIds.length === 0 || !!electionVote.submittedAt,
```

Comment:
IMO making this disabled because of the submittedAt is net negative. If someone finds their way to this page, prevent them from editing, not browsing.

Reply:
Oh, is it because only hitting continue runs the mutation? Hmm. I still don't love it.

Reply:
Yeah I also don't love it, but I think it's an ok compromise given we don't want to spend too much time on this, as disabling every other control individually is a lot more fiddly. I also didn't want to disable the page entirely as people might go back to them to check how they voted